src/commands/create/index.ts

```diff
@@ -182,7 +183,9 @@ export default class Create extends Command {
       num_target_entries: context.targetData.entries.ids.length,
     })
 
-    this.changesetFilePath = path.join(process.cwd(), 'changeset.json')
+    this.changesetFilePath =
+      flags['output-file'] ||
+      path.join(process.cwd(), `changeset-${Date.now()}-${flags.space}-${flags.source}-${flags.target}.json`)
```

Comment:
is this the same structure we use in `contentful-export` and `contentful-import`?

Reply:
yes